msticpy/vis/process_tree.py

```diff
@@ -390,6 +390,8 @@ def _pre_process_tree(
 def _pid_fmt(pid, pid_fmt):
     if pid_fmt == "hex":
         return f"PID: {pid}" if str(pid).startswith("0x") else f"PID: 0x{int(pid):x}"
+    elif pid_fmt == "guid":
```

Comment:
You only need an "if" here since it follows a return statement.
Leaving the elif causes a pylint failure:

```
************* Module msticpy.vis.process_tree
msticpy/vis/process_tree.py:391:4: R[17](https://github.com/microsoft/msticpy/actions/runs/3162919383/jobs/5149980381#step:8:18)05: Unnecessary "elif" after "return", remove the leading "el" from "elif" (no-else-return)



Error: Process completed with exit code 8.
```